docs/web-ui.md

```
@@ -0,0 +1,85 @@
+
+```bash
+calendar-bridge ui -config config.yaml
+```
+
+Then open `http://127.0.0.1:8090`.
+
+If you set an `auth_token`, the browser must present it. The bundled page reads
+the token from the URL fragment (which browsers never send to the server and
+which stays out of server logs):
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)